charts/portal/templates/jobs/cert-update-job.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
     chart: {{ template "portal.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- range $key, $val := .Values.global.additionalLabels }}
+    {{ $key }}: "{{ $val }}"
+    {{- end }}
```

Comment:
note: all three jobs do not have their own values section, so not sure if it needs its own individual label

Reply:
added as portal.jobs.additionalLabels